src/components/content/input-exercise.tsx

```diff
@@ -35,24 +36,54 @@ export function InputExercise({ data }: InputExerciseProps) {
       <Feedback>{feedback}</Feedback>
       <CheckButton
         selectable={value !== ''}
-        onClick={() => setFeedback(checkAnswer(value, data.answers))}
+        onClick={() => setFeedback(checkAnswer())}
       >
         {strings.content.check}
       </CheckButton>
     </Wrapper>
   )
 
-  function checkAnswer(
-    val: string,
-    answers: EdtrPluginInputExercise['state']['answers']
-  ) {
-    const filteredAnswers = answers.filter((answer) => answer.value === val)
+  function checkAnswer() {
+    const answers = data.answers
+    const filteredAnswers = answers.filter((answer) => {
+      const solution = normalize(answer.value)
+      const submission = normalize(value)
+
+      if (data.type === 'input-expression-equal-match-challenge') {
+        return (
+          (solution as A.Expression)
+            .subtract(submission as A.Expression)
+            .toString() === '0'
+        )
+      }
+      return solution === submission
+    })
     if (filteredAnswers.length !== 1 || !filteredAnswers[0].isCorrect) {
       return <span>{strings.content.wrong}</span>
     } else {
       return <span>{strings.content.right}</span>
     }
   }
+
+  function normalize(value: string) {
+    const _value = collapseWhitespace(value)
+    switch (data.type) {
+      case 'input-number-exact-match-challenge':
+        return normalizeNumber(_value).replace(/( )?\/( )?/g, '/')
```

Comment:
@inyono this should be equivalent to the edtr implementation,
but what about `* ` and other options? Wouldn't it make more sense to remove all whitespaces for evaluation? Or is this another edge case?